packages/tldraw/src/lib/ui/hooks/useAnySelectedShapesCount.ts

```diff
@@ -0,0 +1,32 @@
+import { useEditor, useValue } from '@tldraw/editor'
+
+/**
+ * Returns true if the number of LOCKED OR UNLOCKED selected shapes is at least min or at most max.
+ */
+export function useAnySelectedShapesCount(min?: number, max?: number) {
```

Comment:
same comments as `useUnlockedSelectedShapesCount` w.r.t. to naming and simplifying logic